README.md

```diff
@@ -120,19 +121,52 @@ You have two options to install the verifier.
 
 #### Option 1: Install via go
 
-```
+If you want to install the verifier, you can run the following command:
+```bash
 $ go install github.com/slsa-framework/slsa-verifier/v2/cli/slsa-verifier@v2.0.1
 $ slsa-verifier <options>
 ```
 
-#### Option 2: Compile manually
+Tools like [dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates) or [renovate](https://github.com/renovatebot/renovate) use your project's go.mod to identify the version of your dependencies. 
+If you install the verifier in CI, we strongly recommend you follow the steps below to keep the verifier up-to-date:
+
+1. Create a tooling/slsa-verifier.go file containing the following:
```

Comment:
nit: I want to say that folks used to use this pattern with a file name like `*_test.go` for a reason, but it escapes me what that reason is. In any case you probably want to name the file like `tools.go` and that way they can list any CLI tools they need in the same file.

Also, Go doesn't really like file names with hyphen in them, but I don't think it really matters so much in this case.

Reply:
updated the filename.